database/data/schemaHelpers.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../index');
+
+module.exports = {
```

Comment:
Just a suggestion, but completely up to you. I find that defining your export properties/methods within the export object like this to be a little confusing. I think it might be easier to track if you had these functions defined prior to the module.exports assignment. In this way you can then do a very simplistic module.export assignment.

i.e.:
```suggestion
const songSaver = (song, cb) => {
  // some code here
};
module.exports = {
  songSaver,
}
```

Reply:
Thanks for the suggestion.  I like it both ways. I chose my way because it makes it easy for me to clearly see what methods are available on the export object, but yours also achieves that.

Reply:
Got it. Thanks for explaining your reasoning!